src/app/plan/components/Wizard/WizardContainer.tsx

```diff
@@ -26,19 +31,22 @@ export interface IOtherProps {
   isPollingStorage: boolean;
   isPollingClusters: boolean;
   isPollingPlans: boolean;
-  currentPlan: any;
-  currentPlanStatus: any;
+  currentPlan: IPlan;
+  currentPlanStatus: ICurrentPlanStatus;
   startPlanStatusPolling: (planName) => void;
   stopPlanStatusPolling: (planName) => void;
   pvUpdatePollStop: () => void;
   planUpdateRequest: (values, isRerunPVDiscovery) => void;
   resetCurrentPlan: () => void;
   setCurrentPlan: (plan) => void;
   fetchNamespacesRequest: (clusterName) => void;
-  getPVResourcesRequest: () => void;
+  getPVResourcesRequest: (
+    persistentVolumes: IPlanPersistentVolume[],
+    sourceClusterName: IFormValues['sourceCluster']
```

Comment:
![image](https://user-images.githubusercontent.com/11218376/78187705-e8c05500-743c-11ea-994c-61cada6479f9.png)

^^^ so satisfying! Nice job building out the typescript support here.